examples/guides/aggregations/step-2.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+
+    <!-- Include CARTO VL JS -->
+    <script src="../../../dist/carto-vl.js"></script>
+    <!-- Include Mapbox GL JS -->
+    <script src="https://libs.cartocdn.com/mapbox-gl/v0.48.0-carto1/mapbox-gl.js"></script>
+    <!-- Include Mapbox GL CSS -->
+    <link href="https://libs.cartocdn.com/mapbox-gl/v0.48.0-carto1/mapbox-gl.css" rel="stylesheet" />
+    <!-- Make the map visible -->
+    <link rel="stylesheet" type="text/css" href="../../style.css">
+
+    <!-- Numeral.js for number formatting -->
+    <script src="//cdnjs.cloudflare.com/ajax/libs/numeral.js/2.0.6/numeral.min.js"></script>
```

Comment:
I am afraid of using an external library, couldn't we format by creating our own function? I think it adds complexity to the example, although the code looks cleaner. But I would ask for a second opinion, this could be the way to go.

Reply:
hi @VictorVelarde @elenatorro let me read through the guide! i will comment after that :)

Reply:
We're gonna finally keep it